### PR TITLE
Add ignore nan loss and metric

### PR DESCRIPTION
**Pull request by sagi-ezri**
Create classes for metrics and losses in the Keras TensorFlow project that can ignore NaN values in the labels by wrapping another metric or loss and computing it.

### CODE REVIEW

**Comment by gbaned** [comment]
Hi @sagi-ezri It looks like your PR relates to the Keras component. Please submit it to the github.com/keras-team/keras repository instead. Thankyou.
@fchollet, @qlzh727
